integration-test/experimental-react/src/WrappedApolloProvider.tsx

```diff
@@ -1,16 +1,17 @@
 /**
  * implementation to be used with the experimental React hooks implemented in
  * https://github.com/facebook/react/compare/main...phryneas:react:stream-injection
+ *
+ * published under https://www.npmjs.com/package/@phryneas/experimental-react
  */
 
-import React, {
-  useMemo,
-  useActionChannel,
-  useStaticValue,
-  useRef,
-} from "react";
+import {
+  WrapApolloProvider,
+  DataTransportContext,
+} from "@apollo/client-react-streaming";
 import type { DataTransportProviderImplementation } from "@apollo/client-react-streaming";
-import { DataTransportContext } from "@apollo/client-react-streaming";
+import { useMemo, useActionChannel, useStaticValue, useRef } from "react";
+
```

Comment:
Note to self: double check the approach of queueing a single promise for request & response

Reply:
https://github.com/apollographql/apollo-client-nextjs/pull/223